test/src/Provider/GoogleTest.php

```diff
@@ -20,6 +30,15 @@ protected function setUp()
             'hostedDomain' => 'mock_domain',
             'accessType' => 'mock_access_type'
         ]);
+
+        $this->providerOidc = new GoogleProvider([
```

Comment:
Instead of adding this to the existing test, can you make a new one for the different configuration?

Reply:
I've created a separate test class.